psi4/src/core.cc

```diff
@@ -1343,6 +1350,8 @@ PYBIND11_PLUGIN(core) {
         "Assigns the global frequencies to the values stored in the 3N-6 Vector argument.");
     core.def("set_memory_bytes", py_psi_set_memory, py::arg("memory"), py::arg("quiet")=false, "Sets the memory available to Psi (in bytes).");
     core.def("get_memory", py_psi_get_memory, "Returns the amount of memory available to Psi (in bytes).");
+	core.def("set_datadir", &py_psi_set_datadir, "Sets the path to shared text resources, PSIDATADIR");
+	core.def("get_datadir", &py_psi_get_datadir, "Returns the path to shared text resources, PSIDATADIR");
```

Comment:
Did GitHub mess up, or is there a weird indent there? Also, any reason not to do lambda functions again? It seems to increase the readability of these tiny wrapper layers.